.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: osx
   pool:
-    vmImage: macOS-13
+    vmImage: macOS-14
```

Comment:
If you want to persist this across rerenders, you can do
```yaml
azure:
  settings_osx:
    pool:
      vmImage: macos-14
```
in `conda-forge.yml`.

Reply:
No, it was just an experiment